starter-code/javascript/main.js

```diff
@@ -7,25 +7,35 @@ var secDec      = document.getElementById('secDec');
 var secUni      = document.getElementById('secUni');
 var milDec      = document.getElementById('milDec');
 var milUni      = document.getElementById('milUni');
-
-
-function printTime() {
-
-}
+var splitList = document.getElementById('splits');
 
 function printMinutes() {
-
+  console.log(chronometer.twoDigitsNumber(chronometer.getMinutes())[0])
+   minDec.innerHTML = chronometer.twoDigitsNumber(chronometer.getMinutes())[0];
+   minUni.innerHTML = chronometer.twoDigitsNumber(chronometer.getMinutes())[1];
+   console.log(minDec.innerHTML +" "+minUni.innerHTML)
 }
 
 function printSeconds() {
+  secUni.innerHTML = chronometer.twoDigitsNumber(chronometer.getSeconds())[1];
+  secDec.innerHTML = chronometer.twoDigitsNumber(chronometer.getSeconds())[0];
+}
 
+function printTime() {
+  console.log(printMinutes()+" : "+printSeconds())
+  return printMinutes()+" : "+printSeconds()
```

Comment:
The reason why you're getting undefined here is because  your functions printMinutes and printSeconds aren't returning anything, they are just changing the values of the the DOM elements, so by default they are returning undefined. 
So at the end you're returning undefined + undefined.